package.json

```diff
@@ -49,6 +49,10 @@
   "dependencies": {
     "@mapbox/mapbox-gl-directions": "^4.1.0",
     "@mapbox/mapbox-gl-geocoder": "^4.7.4",
+    "@turf/buffer": "^6.5.0",
+    "@turf/collect": "^6.5.0",
+    "@turf/midpoint": "^6.5.0",
+    "@turf/turf": "^6.5.0",
```

Comment:
So these are all for getting the middle of the route? I presume you need all of them. However, you forgot the [types](https://www.npmjs.com/package/@types/turf) :) (devDependencies)

Reply:
no, all is included in turf/turf (also types). I deleted the rest

Reply:
Thanks, I just saw the package is deprecated
cheers!